Dockerfile

```diff
@@ -1,6 +1,9 @@
 # We need the CUDA base dockerfile to enable GPU rendering
 # on hosts with GPUs.
-FROM nvidia/cuda:8.0-devel-ubuntu16.04
+# The image below is a pinned version of nvidia/cuda:9.1-cudnn7-devel-ubuntu16.04 (from Jan 2018)
+# If updating the base image, be sure to test training on vision since it has broken in the past.
+# For example, our custom Tensorflow is compiled to a specific CUDNN minor version.
```

Comment:
Mujoco-py is public, and noone knows what our custom Tensorflow means.

Reply:
changed comment.

Reply:
How would I change the code to use the Dockerfile in CPU instead without CUDA